values.yaml

```diff
@@ -55,19 +55,23 @@ image:
 ##########################
 
 # Number of GraphDB nodes to be deployed as part of the StatefulSet.
-# Set value to 1 to run a standalone GraphDB instance or to more to form a GraphDB cluster.
+# Set value to 1 to run a standalone GraphDB instance or to more to form a GraphDB cluster with GraphDB cluster proxies.
 replicas: 1
```

Comment:
Is this used for the proxy replica count as well?

Reply:
Nope, the proxy has another one `proxy.replicas`